meilisearch/index.py

```diff
@@ -14,22 +14,24 @@ class Index():
 
     config = None
     http = None
-    uid = ""
+    uid = None
+    primary_key = None
 
-    def __init__(self, config, uid):
+    def __init__(self, config, uid, primary_key=None):
         """
         Parameters
         ----------
-        config : Config
-            Config object containing permission and location of meilisearch
+        config : dict
+            Config object containing permission and location of MeiliSearch.
         uid: str
-            Uid of the index on which to perform the index actions.
-        index_path: str
-            Index url path
+            UID of the index on which to perform the index actions.
+        primary_key: str, optional
+            Primary-key of the index.
         """
         self.config = config
-        self.uid = uid
         self.http = HttpRequests(config)
+        self.uid = uid
+        self.primary_key = primary_key
 
     def delete(self):
         """Delete an index from meilisearch
```

Comment:
Same, I plan to do it in another PR, this is not really related to this one since the delete method does not change. I really try to separate the changes to make this PR easier to read. I did it first but it was a mistake : 
https://github.com/meilisearch/meilisearch-python/pull/171#issuecomment-726711808